app/css/popup.css

```diff
@@ -46,9 +88,327 @@ body {
 /**
  * Override Bootstrap responsive because Tab Wrangler is not a mobile UI.
  */
-
 @media (min-width: 576px) {
   .form-inline .form-check {
-    justify-content: flex-start;
```

Comment:
This doesn't need to change, both Firefox and Chrome support `justify-content` without any prefix.